plugins/woocommerce/changelog/57013-fix-45677-tax-rate-api-support-non-latin-class

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix tax rate API support for non-Latin class names.
```

Comment:
Let's keep the empty line at the end.

Reply:
I've made the changelog entry using the GH action from the PR description, should I just add an extra new line there? 🤔